interfaces/builtin/docker_support.go

```diff
@@ -81,6 +81,10 @@ unix (bind,listen) type=stream addr="@/containerd-shim/moby/*/shim.sock\x00",
 /sys/fs/cgroup/*/docker/   rw,
 /sys/fs/cgroup/*/docker/** rw,
 
+# Also allow cgroup writes to kubernetes pods
+/sys/fs/cgroup/*/kubepods/ rw,
+/sys/fs/cgroup/*/kubepods/** rw,
```

Comment:
Is there a reason why these kubernetes specific accesses are in the docker specific interface and not in the kubernetes specific interface? I'm not familiar with the whole picture of how kubernetes is used with strict confinement, but this bit looks like it belongs in something kubernetes specific and not docker specific (as I think kubernetes is a superset of docker, interface wise).

Reply:
These access are in the kubernetes-interface already, but they have to be added to the docker-support interface because while kubelet drives docker (docker is one of the container runtime backends), docker then needs to work with the cgroups kubelet setup.